homeassistant/components/mystrom/sensor.py

```diff
@@ -0,0 +1,95 @@
+"""Support for myStrom sensors of switches/plugs."""
+from __future__ import annotations
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import (
+    PLATFORM_SCHEMA,
+    SensorDeviceClass,
+    SensorEntity,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_NAME, UnitOfPower, UnitOfTemperature
+from homeassistant.core import HomeAssistant
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN, MANUFACTURER
+
+DEFAULT_NAME = "myStrom Switch"
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_HOST): cv.string,
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    }
+)
```

Comment:
This is not needed anymore since this part will be config entry only

Reply:
Done, this could be removed in the switch.py too then?

Reply:
Nope, that's deprecated and will be removed in the December release if I recall correctly